ansible/tasks/oauth2-proxy.yml

```
@@ -1,9 +1,22 @@
-- name: "download oauth2_proxy"
+---
+- set_fact:
+  oauth_version: "oauth2_proxy-v5.0.0.linux-amd64.go1.13.6"
```

- We probably also want the version to be a variable
- This is a download filename, so we need to append .tar.gz (if I'm not mistaken)

```suggestion
  oauthproxy_filename: "oauth2_proxy-v5.0.0.linux-amd64.go1.13.6.tar.gz"
  oauthproxy_version: "v5.0.0"
  oauthproxy_checksum: "sha256:8aa23a0635fe9ee5406e1cbef1fb86de44c660ff2269e05a04294989121c2408"
```